packages/view-types/feature-list/src/selectable-table/SelectableTable.js

```diff
@@ -178,22 +196,25 @@ export default function SelectableTable(props) {
           />
         </label>
       </div>
-      {columns.map(column => (
+      {columns.map((column, columnIndex) => {
+        const columnClass = columnIndex === 0 ? classes.tableCellFirst : classes.tableCellSecond;
+        return (
         <div
-          className={classes.tableCell}
+          className={`${classes.tableCell} ${columnClass}`}
           key={column}
         >
           {data[index][column]}
         </div>
-      ))}
+      )})}
     </div>
   );
 
   const headerRowRenderer = ({ style }) => (
-    <div className={clsx({ [classes.hiddenInputColumn]: !showTableInputs }, classes.tableRow)} style={style}>
```

Comment:
I don't get the point of `{clsx({ [classes.hiddenInputColumn]: !showTableInputs }` <-- I removed it, because while it was there, it was making the elements hidden all the time.

Reply:
This is fine with me! I believe `showTableInputs` is not currently used